### PR TITLE
Homepage content update

### PR DESCRIPTION
**Pull request by RenDavidDimen**
# Summary | Résumé

Content update for homepage strings, its content, section ordering, and image references. Static image files are uploaded to the Staging S3 bucket, will need to upload them to S3 Production should the review be successful

Trello Card: https://trello.com/c/gEGgx51w/204-update-homepage-content

### CODE REVIEW

File: app/templates/views/signedout.html [hunk 1]
```diff
@@ -164,23 +164,23 @@
   <!-- end header -->
   <div id="content" class="home-page-content contain-floats">
   <!-- start section 1 -->
-  <div class="bg-blue-backgr" role="main">
+  <div class="bg-blue" role="main">
```
Comment (AntoineAugusti) [inline app/templates/views/signedout.html:167]: I think the blue colour `backgr` was introduced just for this page. Can you double-check it's not used anymore and then delete it here?

https://github.com/cds-snc/notification-admin/blob/c920b0968e221044e7c57bf6387e7a4e033d6344/tailwind.config.js#L57

File: app/templates/views/signedout.html [hunk 2]
```diff
@@ -218,176 +218,104 @@ <h2 class="leading-tight font-sans font-bold text-lg lg:hidden">
...
-         {% endif %}
-         </div>
           <ul class="list-disc ml-5">
             <li class="pt-4 text-base leading-tight">
-              {{ _("Staff who regularly write and send service updates wishing to upgrade the tools used to communicate with the public") }}
+              {{ _("Send consistent and accessible email messages from a trusted domain.") }}
             </li>
             <li class="pt-4 text-base leading-tight">
-              {{ _("Developers working on improving workflow and tooling for public servants") }}
+              {{ _("Personalise your messages, while protecting  personal information.") }}
```
Comment (AntoineAugusti) [inline app/templates/views/signedout.html:243]: You've got two spaces before "personal information"

File: app/templates/views/signedout.html [hunk 3]
```diff
@@ -218,176 +218,104 @@ <h2 class="leading-tight font-sans font-bold text-lg lg:hidden">
...
               loading="lazy"
               width="400"
-              src="{{ asset_s3_url('text-messages-fr.svg') }}"
+              src="{{ asset_s3_url('schedule-email-fr.svg') }}"
             />
           {% endif %}
 
           <div class="text-small">
-            {{ _("Create templates and save them for future messages") }}
+            {{ _("Schedule and send up to 50,000 messages to a list of recipients") }}
```
Comment (AntoineAugusti) [inline app/templates/views/signedout.html:313]: You can use `CSV_MAX_ROWS` in the config for this number.

Use a `{}` in your string and the helper `format_number` to deal with thousands separator in French/English

File: app/templates/views/signedout.html [hunk 4]
```diff
@@ -419,198 +347,281 @@ <h2 class="leading-tight font-bold font-sans text-lg pb-6">
...
+  <!-- end section 4 -->
 
-    <!-- start section 6 -->
-    <div class="bg-white">
-      <div class="container px-10 py-24">
+  <!-- start section 5 -->
+  <div class="bg-gray">
+    <div class="container px-10 py-24">
+      <div class="flex flex-col lg:flex-row justify-between">
+        <!-- -->
```
Comment (AntoineAugusti) [inline app/templates/views/signedout.html:437]: Remove?

File: app/templates/views/signedout.html [hunk 5]
```diff
@@ -407,10 +335,10 @@ <h2 class="leading-tight font-bold font-sans text-lg pb-6">
...
 
           <div class="text-small">
-            {{ _("Track the status of the messages you send") }}
+            {{ _("Track message delivery status in your dashboard") }}
           </div>
         </div>
-        <!-- end note box -->
+        <!-- end note box 3 -->
 
         <!-- -->
```
Comment (AntoineAugusti) [inline app/templates/views/signedout.html:343]: Remove?

File: app/templates/views/signedout.html [hunk 6]
```diff
@@ -419,198 +347,281 @@ <h2 class="leading-tight font-bold font-sans text-lg pb-6">
...
+      <h2 class="leading-tight font-bold lg:hidden font-sans text-lg pb-6">
+        {{ _("Skip the procurement process") }}
+      </h2>
+      <div class="flex flex-col lg:flex-row justify-between">
+        <div class="text-content max-w-4xl">
+          <h2 class="leading-tight font-bold hidden lg:block lg:visible font-sans text-lg">
+            {{ _("Skip the procurement process") }}
+          </h2>
+          <p class="pt-4 text-base mb-0">
+            {{ _("GC Notify is managed by the Canadian Digital Service  with no costs for departments that use the service.") }}
```
Comment (AntoineAugusti) [inline app/templates/views/signedout.html:525]: two spaces before "with no costs"

File: app/templates/views/signedout.html [hunk 7]
```diff
@@ -419,198 +347,281 @@ <h2 class="leading-tight font-bold font-sans text-lg pb-6">
...
+          </p>
+          <ul class="list-disc ml-10">
+            <li class="pt-4 text-base mb-0">
+              {{ _("Replace paper letters or costly third-party messaging services.")
+              }}
+            </li>
+            <li class="pt-4 text-base mb-0">
+              {{ _("Send up to 10 million emails per year and up to 25,000 text messages per year, completely free.")}}
```
Comment (AntoineAugusti) [inline app/templates/views/signedout.html:533]: Should be able to use the config `DEFAULT_FREE_SMS_FRAGMENT_LIMITS` with `central`. I think it's already used somewhere in the codebase, you can do a quick search

File: app/templates/views/signedout.html [hunk 8]
```diff
@@ -419,198 +347,281 @@ <h2 class="leading-tight font-bold font-sans text-lg pb-6">
...
-          </div>
-          <!-- Box 3-->
-          <div class="pb-10 lg:pb-0">
-            <div class="text-xxl font-bold leading-none font-sans">{{ stats.notifications_total | format_number }}</div>
-            <div class="text-base py-2">{{ _("messages sent") }}</div>
+  </div>
+  <!-- end section 6 -->
+
+  <!-- start section 7 -->
+  <!-- end section 7 -->
```
Comment (AntoineAugusti) [inline app/templates/views/signedout.html:570]: ?

**Review by AntoineAugusti — CHANGES_REQUESTED** [review]
Good work! A few suggestions. I'll let you fix the footer colour as well to see links.

Can #864 be closed in favour of this one?

File: app/templates/views/signedout.html (hunk 2 above)
Comment (RenDavidDimen) [reply app/templates/views/signedout.html:243]: Good catch! 🕵️‍♂️

File: app/templates/views/signedout.html (hunk 1 above)
Comment (RenDavidDimen) [reply app/templates/views/signedout.html:167]: Can confirm that it is no longer being used 👍

**Review by anikbrazeau — CHANGES_REQUESTED** [review]
This is great! Thanks for making it happen! 👍 

Three small things:

- Change link destination for "How GC Notify meets government needs" / "Comment GC Notification répond aux besoins du gouvernement" to go to "Why GC Notify" page
- Change FR equivalent of "Skip the procurement process" to "Nul besoin d'approvisionnement"
- Can we upload a new first image for the personalised template to match Figma (new illustration shows transactional example and not text message specific)

File: app/translations/csv/fr.csv [hunk 9]
```diff
@@ -614,29 +614,31 @@
...
+"Personalise your messages, while protecting personal information.","Personnalisez vos messages tout en protégeant les renseignements personnels."
+"Reduce the time, effort, and cost of correspondence.","Réduisez le temps, les efforts et les coûts associés à la correspondance."
+"How GC Notify meets government needs","Comment GC Notification répond aux besoins du gouvernement"
+"Work more efficiently","Travaillez plus efficacement"
+"Automate sending so that whenever there is a status change, the public receives an update.","Automatisez vos envois de sorte que dès qu'il y a un changement de statut, le public reçoit une mise à jour."
+"Improve your development tools with a secure, cloud-based messaging platform.","Améliorez vos outils grâce à une plateforme de messagerie infonuagique sécurisée."
+"Set permission access for which people on your team can edit or send messages.","Contrôlez les droits d’accès des membres de votre équipe pour envoyer ou modifier des messages."
+"What you can do with GC Notify","Avec GC Notification, vous pouvez :"
```
Comment (RenDavidDimen) [inline app/translations/csv/fr.csv:628]: Found another string with a different translation. Currently talking with design to see if we can find a solution to this

File: app/main/views/index.py [hunk 10]
```diff
@@ -45,6 +45,8 @@ def index():
         scrollTo="false",
         admin_base_url=current_app.config['ADMIN_BASE_URL'],
         stats=get_latest_stats(get_current_locale(current_app)),
+        csv_max_rows=current_app.config['CSV_MAX_ROWS'],
+        default_free_sms_fragment_limits_central=current_app.config.get('DEFAULT_FREE_SMS_FRAGMENT_LIMITS', {}).get('central', 10000),
```
Comment (AntoineAugusti) [inline app/main/views/index.py:49]:
```suggestion
        default_free_sms_fragment_limits_central=current_app.config['DEFAULT_FREE_SMS_FRAGMENT_LIMITS']['central'],
```
Don't want to fallback to another value if this config dict gets updated, we would need to change code here and this is expected

**Review by AntoineAugusti — APPROVED** [review]
LGTM 🚢  Good work 🥳 

Before merging, add cache headers in staging, upload files in prod, set cache headers as discussed on Slack.

**Review by anikbrazeau — APPROVED** [review]
Looks great! Thanks for your patience on this one!

**Pull request merged**